controllers/utils/utils.go

```diff
@@ -19,8 +19,6 @@ import (
 	"k8s.io/client-go/discovery"
 	"k8s.io/client-go/rest"
 	"sigs.k8s.io/controller-runtime/pkg/client"
-
-	"github.com/opendatahub-io/odh-model-controller/controllers/constants"
```

Comment:
I think you need to add this back

Reply:
The import was no longer being used in the file -- still should be added back?

Reply:
I think after rebasing on the latest master it will need the import again

Reply:
thanks for this! I've rebased and pulled all the recent changes now. :)